pkg/cmd/enable/addons/exec.go

```diff
@@ -0,0 +1,156 @@
+// Copyright Contributors to the Open Cluster Management project
+package addons
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	clusterclientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+	"open-cluster-management.io/clusteradm/pkg/cmd/enable/addons/scenario"
+	"open-cluster-management.io/clusteradm/pkg/helpers"
+	"open-cluster-management.io/clusteradm/pkg/helpers/apply"
+)
+
+//ClusterName: The cluster name used in the template
+type ClusterName struct {
+	ClusterName string
+}
+
+func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
+	klog.V(1).InfoS("addon options:", "dry-run", o.ClusteradmFlags.DryRun, "names", o.names, "clusters", o.clusters, "output-file", o.outputFile)
+
+	if err := o.validate(); err != nil {
```

Comment:
The validate is automatically called by the cmd definition. I think the lines below the validate should go in the run

Reply:
done as suggested